db/migrate/20200328120326_remove_member_type_from_family.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveMemberTypeFromFamily < ActiveRecord::Migration[6.0]
+  def change
+    remove_column :families, :member_type, :integer
+    remove_column :families, :owner, :integer
+  end
+end
```

Comment:
due to my comment above, this migration will no longer needed

Reply:
removed it